### PR TITLE
Add new feature to assign coating method. User can specify an object or face ID

### LINKED ISSUE #2070: Assign coating to faces
### Description of the feature

Right now it is only possible to assign finite conductivity boundary to objects, but it is also possible to assign to Faces.

### Steps for implementing the feature

Add the option to add faces to Finite Conductivity boundary

### Useful links and references

_No response_

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**
Close #2070

### CODE REVIEW

File: pyaedt/hfss.py [hunk 1]
```diff
@@ -656,20 +656,42 @@ def assign_coating(
         Examples
         --------
 
-        Create a cylinder at the XY working plane and assign a copper coating of 0.2 mm to it.
-
+        Create two cylinders at the XY working plane and assign a copper coating of 0.2 mm to the inner cylinder and
```
Comment (MaxJPRey) [inline pyaedt/hfss.py:659]:
```suggestion
        Create two cylinders in the XY working plane and assign a copper coating of 0.2 mm to the inner cylinder and
```

File: pyaedt/modeler/cad/Primitives3D.py [hunk 2]
```diff
@@ -585,6 +585,15 @@ def create_rectangle(self, csPlane, position, dimension_list, name=None, matname
...
         >>> oEditor.CreateRectangle
 
+        Examples
+        --------
+        >>> from pyaedt import Hfss
+        >>> hfss = Hfss()
+        >>> plane = 0
+        >>> position = [0,0,0]
+        >>> dimension = [10,5]
+        >>> rect_obj = hfss.modeler.create_rectangle(plane, position, dimension)
```
Comment (MaxJPRey) [inline pyaedt/modeler/cad/Primitives3D.py:595]: Thanks @Samuelopez-ansys for adding some examples. This is great.

**Review by MaxJPRey — APPROVED** [review]

File: pyaedt/hfss.py [hunk 3]
```diff
@@ -696,7 +718,7 @@ def assign_coating(
             props["IsShellElement"] = issheelElement
         else:
             props["IsInternal"] = isInternal
-        return self._create_boundary("Coating_" + listobjname[:32], props, "Finite Conductivity")
+        return self._create_boundary("Coating_" + listobjname[1:32], props, "Finite Conductivity")
```
Comment (MaxJPRey) [inline pyaedt/hfss.py:721]: For my knowledge why is it `[1:32]`? Why 32?

Comment (Samuelopez-ansys) [reply pyaedt/hfss.py:721]: @MaxJPRey that's a good question, I think it is a legacy convention to have less than 32 characters, @maxcapodi78 @Alberto-DM do you know if the limit of 32 is cosmetic or not?

Comment (Samuelopez-ansys) [inline pyaedt/hfss.py:721]:
```suggestion
        return self._create_boundary("Coating_" + listobjname[1:], props, "Finite Conductivity")
```

**Pull request merged**